src/main/java/com/icthh/xm/uaa/service/dto/UserDTO.java

```diff
@@ -97,6 +97,15 @@ public class UserDTO {
 
     private Instant lastLoginDate;
 
+    /**
+     * User has set password during registration himself
+     */
+    private Boolean passwordSetByUser;
```

Comment:
do we need this fields in UserDTO? it's can be change only by logic, not by end-user

Reply:
fixed